### PR TITLE
edited team page, pricing page

### PR DESCRIPTION
**Pull request by eggchul**

### CODE REVIEW

File: tests/unit/example.spec.js [hunk 1]
```diff
@@ -1,12 +1,33 @@
...
+// import HelloWorld from '@/components/HelloWorld.vue'
+//
+// describe('HelloWorld.vue', () => {
+//   it('renders props.msg when passed', () => {
+//     const msg = 'new message'
+//     const wrapper = shallowMount(HelloWorld, {
+//       propsData: { msg }
+//     })
+//     expect(wrapper.text()).toMatch(msg)
+//   })
```
Comment (bhutaninikhil1991) [inline tests/unit/example.spec.js:11]: why do we have commented out code here?

File: tests/unit/example.spec.js [hunk 2]
```diff
@@ -1,12 +1,33 @@
...
+  //     wrapper.setData({ x1: 5, x2: 10 });
+  //     await wrapper.vm.$nextTick();
+  //     expect(wrapper.text()).toContain("10");
+  // });
+
+  // it("button click without correct sum", () => {
+  //     expect(wrapper.vm.message).toBe("");
+  //     const button = wrapper.find("button");
+  //     button.trigger("click");
+  //     expect(wrapper.vm.message).toBe("TRY AGAIN");
```
Comment (bhutaninikhil1991) [inline tests/unit/example.spec.js:24]: same here

**Review by bhutaninikhil1991 — APPROVED** [review]
rest Looks good

File: tests/unit/example.spec.js (hunk 1 above)
Comment (eggchul) [reply tests/unit/example.spec.js:11]: this one is an example script for review and future implementation

**Pull request merged**